app/views/applications/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1><%= @app.name%>'s Application </h1>
+
+<ul>
+  <li>Name: <%= @app.name %></li>
+  <li>Address: <%= @app.address %></li>
+  <li>City: <%= @app.city %></li>
+  <li>State: <%= @app.state %></li>
+  <li>Zip: <%= @app.zip %></li>
+  <li>Phone Number: <%= @app.phone_number %></li>
+  <p>Description: <%= @app.description %></p>
+</ul>
+
+  <h3>Pets Applied For:</h3>
+  <p>Check each pet you'd like to approve. Pending pets cannot be approved.</p>
+  <%= form_tag ("/applications/#{@app.id}") do %>
+    <% @apps_pets.each do |app_pet| %>
+    <% pet = Pet.get_pet_by_id(app_pet.pets_id) %>
+      <ul>
+        <% if pet.adoption_status == "Adoptable" %>
+        <li id=<%="checked_field-#{pet.id}"%>><%= link_to "#{pet.name}", "/pets/#{pet.id}" %> - <%= check_box_tag "#{pet.id}", "#{pet.name}"%></li>
+        <% else %>
+          <li><%= link_to "#{pet.name}", "/pets/#{pet.id}" %> - This pet is pending adoption. - <%= link_to "Revoke Application", "/applications/#{@app.id}/#{pet.id}", method: :patch %></li>
+        <% end %>
+      </ul>
+    <% end %>
+    <li><%= submit_tag "Approve Application" %></li>
+  <% end %>
```

Comment:
Awesome. I see we can approve an app for multiple pets and then revoke an application for a specific pet